cmd/kubeadm/app/cmd/init.go

```diff
@@ -218,10 +218,10 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight,
 	if cfgPath != "" {
 		b, err := ioutil.ReadFile(cfgPath)
 		if err != nil {
-			return nil, fmt.Errorf("unable to read config from %q [%v]", cfgPath, err)
+			return nil, fmt.Errorf("Unable to read config from %q [%v]", cfgPath, err)
```

Comment:
errors are lower case

Reply:
i think i made a lot of these uppercase, do you want them all reverted to lowercase, or does this only apply to those which you commented in your review? thanks.